pkg/archive/changes.go

```diff
@@ -359,7 +359,7 @@ func ChangesSize(newDir string, changes []Change) int64 {
 }
 
 // ExportChanges produces an Archive from the provided changes, relative to dir.
-func ExportChanges(dir string, changes []Change) (Archive, error) {
+func ExportChanges(dir string, changes []Change, excludes []string) (Archive, error) {
```

Comment:
@vbatts If add a new function I thinks I'll do it like this: 
1. Rename the current `ExportChanges` to `exportChangesImpl`. 
2. Call `exportChangesImpl` in `ExportChanges(dir string, changes []Change)` with an empty string slice.
3. Add a new function `ExportChanges(dir string, changes []Change, excludes []string)` to call `exportChangesImpl` directly.

Please let me know if it makes sense. Thanks.